app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Great work seting up devise however the views for login and signup pages should look like the provided wireframes, kindly make sure to implement this so your app looks like the  provided wireframes

||||
|-|-|-|
|![Screenshot_20230201_183828](https://user-images.githubusercontent.com/25227022/216120569-6fefb9c9-1b08-45cf-b896-4df1596e5ae9.png)|![Screenshot_20230201_183911](https://user-images.githubusercontent.com/25227022/216120600-bfa968f8-2acb-4150-a35f-01474bf044bf.png)|![image](https://user-images.githubusercontent.com/25227022/216120680-c0e54d5e-fa46-444f-834b-cf7ea273b9e9.png)|